lib/elasticband/aggregation/terms.rb

```diff
@@ -5,7 +5,7 @@ class Terms < Aggregation
 
       def initialize(name, field, options = {})
         super(name)
-        self.field = field.to_sym
+        self.field = field.to_sym if field
```

Comment:
plain ruby, it's faster and simpler. I prefer `try` when you have multiple things going on